### ISSUE TITLE
Load all tokens on the stats page

### ISSUE THREAD

**Issue by shanefontaine** [hop-protocol/hop#527]
There are times when the stats page does not fully load all tokens on the "Debit Window Stats" table. All the tokens should be loaded

Goal

* Display all tokens on every load

Considerations

* I'm not quite sure how to reliably reproduce this 🤔 

![Image](https://user-images.githubusercontent.com/9441295/277524934-3e412675-9b43-4ba6-9f34-3ce462594862.png)

**Comment by miguelmota** [comment]
The reason it wasn't showing some rows was due to rpc errors so it would not display anything for those items. 

I made an [update](https://github.com/hop-protocol/hop/commit/c02bc1bbf73035f058773559125c2430ebe5afad) so it at least shows the empty row and shows the error message, if there's an error when fetching any item for any table.

Example error display:

![DeepinScreenshot_select-area_20231107184306](https://github.com/hop-protocol/hop/assets/168240/9dd08357-1094-4a1b-9e13-128a35e21c00)

The typical RPC error seen was that it was exceeding the rate limit retries so ethers throws 'missing revert data' generic error.
